src/crs.rs

```diff
@@ -14,7 +14,12 @@
 
 use std::collections::BTreeMap;
 
-use rustc_serialize::json::{self, ToJson};
+#[cfg(not(feature = "with_serde"))]
```

Comment:
Is there any reason why we should not allow `rustc_serialize` to work alongside `serde`?